build/configs/rtl8730e/Make.defs

```diff
@@ -219,7 +219,7 @@ endef
 SECURE_HEADER_SIZE=32
 
 define MAKE_SAMSUNG_HEADER
-  $(TOPDIR)/../os/tools/mksamsungheader.py kernel ${SECURE_HEADER_SIZE}
+  $(TOPDIR)/../os/tools/mksamsungheader.py $(1) kernel ${SECURE_HEADER_SIZE}

```

Comment:
@ewoodev Could you confirm this is enough to resolve the build break?
@zhongnuo-tang This is not `update peripherals`. Could you split the commit for this?

Reply:
Dear @sunghan-chang,

Okay, I will split the commit after @ewoodev has confirmed that it can fix the build break.

Reply:
Dear @ewoodev,
Should i commit this fix in this PR or Samsung will fix this?

Dear @sunghan-chang,
If I would need to commit this fix, should it be another commit in this same PR or raise another PR for this fix?

Thank you

Reply:
@zhongnuo-tang It should be a different commit but both is ok for a PR.